app/src/buildconfig.ts

```diff
@@ -288,12 +246,12 @@ function get_bugsnag_key(): string | false {
   return bugsnag_key;
 }
 
-function get_conductor_url(): string {
-  const url = import.meta.env.VITE_CONDUCTOR_URL;
-  if (url) {
-    return url;
+function get_conductor_urls(): string[] {
+  const urls = import.meta.env.VITE_CONDUCTOR_URL;
+  if (urls) {
+    return urls.split(' ');
```

Comment:
space is risky separator due to trailing whitespace - would propose comma as a more standard separator, and trim/strip each split value for whitespace, filtering out empty values.

Reply:
Hmm, I had a comma but I thought they are valid in a URL I chose space instead, seems like I was wrong, comma is not a valid char in a URL so we can use that instead.